docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,353 @@
 ---
 title: Setting Up a Flow Access Node
 sidebar_title: Access Node Setup
-description: How to run a Permissionless Access node
+description: How to run a permissionless Access node
 ---
 
-This guide is for running a Permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+This guide is for running a permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+
+Permissionless Access nodes allow any operator to run a Flow Access node.
+Unlike the other staked nodes, a permissionless access node does not have to be approved by the service account before it can join the network, hence the term "permissionless". The goal is to make all node types permissionless and this is the first step towards achieving that goal.
+
+## Who should run a Permissionless Access node?
+dApp developers can choose to run their own private permissionless access node and move away from using the community access nodes. This will also allow them to not be subjected to the API rate limits of the public access nodes.
+
+Node operators can also run their own permissionless access node and provide access to that node as a service.
+
+Chain analytics, audit and exploration applications can run such an access node and do not have to rely on third parties for the state of the network.
+
+
+## Timing
+
+New nodes are able to join the network each time a new epoch begins.
+An epoch is a period of time (approximately one week) when the node operators in the network are constant.
+At epoch boundaries, newly staked node operators are able to join the network and existing node operators which have unstaked may exit the network.
+You can read more about epochs [here](https://developers.flow.com/nodes/staking/schedule).
+
+
+In order to join the network at epoch N+1, the access node **must** be registered with at least 100 FLOW staked prior to the end of epoch N's Staking Auction Phase.
+
+Currently on mainnet, the staking auction starts every Wednesday at around 17:30 UTC and ends on the next Wednesday at 8:30 UTC.
+Hence, the node should be staked by _Wednesday, 08:30 UTC_ to be able to join the network in the next epoch.
+
+Confirmation of a new node's inclusion in epoch N+1 is included in the [`EpochSetup` event](/staking/epoch-scripts-events/#epochsetup).
+
+
+![Flow Epoch Schedule](https://storage.googleapis.com/flow-resources/documentation-assets/epoch-startup-order.png)
+
+## Limitations
```

Comment:
I will be revising this section after 15th Feb based on what happens. If all slots are taken then I will indicate that.

Reply:
done